packages/components/src/components/post-rating/post-rating.scss

```diff
@@ -0,0 +1,98 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+:host {
+  display: inline-block;
+}
+
+.rating {
+  margin: 0 (-(post.$size-micro));
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  overflow: hidden;
+  & :focus {
+    outline: none;
+  }
+  &[aria-readonly='true'] .star-container {
+    cursor: default;
+  }
+}
+
+.rating:focus-visible {
+  outline-offset: post.$input-focus-outline-thickness;
+  outline: post.$input-focus-outline-thickness solid post.$input-focus-color;
+  position: relative;
+  z-index: 1;
+}
+
+.star-container {
+  position: relative;
+  display: block;
+  width: post.$lead-font-size;
+  height: post.$lead-font-size;
```

Comment:
Why not using the post.$size-{x} variables? Or just add `1em`, as this is the current outputted value.

Reply:
I could also just use 1em, the goal is that the stars have the same size as the text.